src/Api/User.hs

```diff
@@ -1,28 +1,27 @@
-{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TypeOperators     #-}
+{-# LANGUAGE TypeOperators #-}
 
 module Api.User where
 
-import           Control.Monad.Except        (MonadIO, liftIO)
-import           Control.Monad.Logger        (logDebugNS)
-import qualified Control.Monad.Metrics       as Metrics
-import           Data.Int                    (Int64)
-import           Database.Persist.Postgresql (Entity (..), fromSqlKey, insert,
-                                              selectFirst, selectList, (==.))
-import           Servant
-import           Servant.JS                  (vanillaJS, writeJSForAPI)
+import Control.Monad.Except (MonadIO, liftIO)
+import Control.Monad.Logger (logDebugNS)
+import qualified Control.Monad.Metrics as Metrics
+import Data.Int (Int64)
+import Database.Persist.Postgresql
+       (Entity(..), fromSqlKey, insert, selectFirst, selectList, (==.))
```

Comment:
If the import and module goes beyond the column count, then the import list is indented, but is kept on one line. This keeps the import lists compact in the smallest cases, where it's easier to notice a small change.